code/game/objects/effects/portals.dm

```diff
@@ -40,10 +40,11 @@
 
 /obj/effect/portal/attackby(obj/item/W, mob/user, params)
 	if(user && Adjacent(user))
+		user.forceMove(get_turf(src))

```

Comment:
Why is this forceMove being added exactly? This will end up calling `teleport` twice.

Reply:
uhhh hold on im gonna go look something up i believe this was supposed to fix something hm